### PR TITLE
whitelist amp-list for amp-sidebar

### PR DESCRIPTION
**Pull request by zhouyx**
simply whitelist `amp-list` for `amp-sidebar`.

### CODE REVIEW

File: test/manual/amp-sidebar-amp-list-test.html [hunk 1]
```diff
@@ -0,0 +1,506 @@
+<!doctype html>
```
Comment (sriramkrish85) [inline test/manual/amp-sidebar-amp-list-test.html]: delete this new file. add this amp-list (may be more than 1 amp-list) to the bottom of the existing amp-sidebar example

File: extensions/amp-list/0.1/validator-amp-list.protoascii [hunk 2]
```diff
@@ -48,7 +48,6 @@ tags: {  # amp-list
 }
 tags: {  # <amp-list>
   tag_name: "AMP-LIST"
-  disallowed_ancestor: "AMP-SIDEBAR"
```
Comment (sriramkrish85) [inline extensions/amp-list/0.1/validator-amp-list.protoascii]: @Gregable is this the only change that is required?

File: test/manual/amp-sidebar-amp-list-test.html (hunk 1 above)
Comment (zhouyx) [inline test/manual/amp-sidebar-amp-list-test.html]: done

File: test/manual/amp-sidebar.amp.html [hunk 3]
```diff
@@ -237,6 +239,18 @@
...
+    <amp-list width="396" height="80" layout=fixed
+      src="/test/manual/amp-list-data.json?RANDOM">
+      <template type="amp-mustache">
+        <div class="story-entry">
+          <amp-img src="{{imageUrl}}" width=80 height=60></amp-img>
+          {{title}}
+        </div>
+      </template>
+      <div overflow>
+        SEE MORE
```
Comment (sriramkrish85) [inline test/manual/amp-sidebar.amp.html]: See More

**Comment by sriramkrish85** [comment]
LGTM - waiting for @Gregable to take a look

File: test/manual/amp-sidebar.amp.html (hunk 3 above)
Comment (zhouyx) [inline test/manual/amp-sidebar.amp.html]: Copy that from amp-list example. we don't want that here?

File: extensions/amp-list/0.1/validator-amp-list.protoascii (hunk 2 above)
Comment (Gregable) [inline extensions/amp-list/0.1/validator-amp-list.protoascii]: Yes.

**Comment by sriramkrish85** [comment]
@Gregable @powdercloud can you take a look?

**Comment by Gregable** [comment]
I already commented on this, but yes, this is fine.

**Pull request merged**